feature-libs/my-account/organization/occ/converters/occ-org-unit-address-list-normalizer.ts

```diff
@@ -1,27 +1,27 @@
 import { Injectable } from '@angular/core';
-import { B2B_ADDRESS_NORMALIZER } from '@spartacus/my-account/organization/core';
 import {
+  Address,
+  ADDRESS_NORMALIZER,
   Converter,
-  Occ,
-  EntitiesModel,
-  B2BAddress,
   ConverterService,
+  EntitiesModel,
+  Occ,
 } from '@spartacus/core';
 
 @Injectable()
 export class OccOrgUnitAddressListNormalizer
-  implements Converter<Occ.B2BAddressList, EntitiesModel<B2BAddress>> {
+  implements Converter<Occ.B2BAddressList, EntitiesModel<Address>> {
```

Comment:
We can also unify `Occ.B2BAddressList` with `Occ.AddressList` and `Occ.B2BAddress` with `Occ.Address`.
Need to only add to Address model.
```
cellphone?: string;
district?: string;
```